blink/handout.tex

```diff
@@ -39,13 +39,6 @@
 
 \setlist[enumerate,1]{start=0}
 
-\newcommand\plpage{
-    \newpage
-    \begin{tikzpicture}[remember picture,overlay]
-        \node[opacity=0.1,below left,xshift=1cm,yshift=1cm] at (current page.north east) {\includegraphics[width=12cm]{../logo}};
```

Comment:
I guess we can now delete the logo entirely.

Reply:
AFAICS, it's not in the repo in the first place

Reply:
heh, it's ignored because it's pdf ;)